### PR TITLE
[WIP] Danger section

### PR DESCRIPTION
**Pull request by sapegin**
* [x] Comment screenshot
* [x] GitHub bot
* [x] More API methods (not just `warn`)
* [x] More examples

### CODE REVIEW

File: manuscript/infrastructure/03-automation.md [hunk 1]
```diff
@@ -191,9 +191,67 @@ https://www.npmjs.com/package/gh-lint
...
+
+```js
+import { danger, warn } from 'danger';
+
+// Warn if the PR has changes in package.json but no changes in package-lock.json
+const packageChanged = danger.git.modified_files.includes('package.json');
+const lockfileChanged = danger.git.modified_files.includes('package-lock.json');
+if (packageChanged && !lockfileChanged) {
+  warn(`Changes were made to package.json, but not to package-lock.json.
+Perhaps you need to run \`npm install\` and commit changes in package-lock.json.. Make sure you’re using npm 5+.`);
```
Comment (bebraw) [inline manuscript/infrastructure/03-automation.md]: Line length might be an issue here. I've found 68 characters fit comfortably. Anything over that and you get a newline in the output. Maybe something to lint for the code examples.

File: manuscript/infrastructure/03-automation.md [hunk 2]
```diff
@@ -191,9 +191,67 @@ https://www.npmjs.com/package/gh-lint
...
+```yaml
+language: node_js
+node_js:
+  - 8
+script:
+  npm run test
+  npm run danger
+```
+
+Now every time someone sends a pull request that changes *package.json* but not *pakcage-lock.json*, bot will warn them:
```
Comment (bebraw) [inline manuscript/infrastructure/03-automation.md]: /s/pakcage-lock.json/package-lock.json. Add a textlint rule for this.

File: manuscript/infrastructure/03-automation.md (hunk 1 above)
Comment (sapegin) [reply manuscript/infrastructure/03-automation.md]: Can we configure Prettier to do that? ;-)

File: manuscript/infrastructure/03-automation.md (hunk 2 above)
Comment (sapegin) [reply manuscript/infrastructure/03-automation.md]: 👍

File: manuscript/infrastructure/03-automation.md (hunk 1 above)
Comment (bebraw) [reply manuscript/infrastructure/03-automation.md]: Yup, `"printWidth": 70` or so.

**Comment by sapegin** [comment]
I think it’s good enough now.

**Pull request merged**